snooty/taxonomy.toml

```diff
@@ -4,55 +4,221 @@ name = "reference"
 [[genres]]
 name = "tutorial"
 
+# trying to map to docs "name" property 
+# for target_platforms and sub_platforms
+# as much as possible
 [[target_platforms]]
 name = "atlas"
-[[target_platforms.versions]]
-name = "v1.2"
-[[target_platforms.versions]]
-name = "master"
+
+  [[target_platforms.sub_platforms]]
+  name = "atlas-app-services"
+  display_name = "App Services"
```

Comment:
I was thinking that for these subproducts, since they'll all also have the higher level "Atlas" tag, we could take atlas off the name.  Or is the name value coming from how we differentiate between them in the repos?

Reply:
Except for Atlas CLI, which doesn't make sense without Atlas in front.

Reply:
most names are an attempt to map to a snooty project name ie. [atlas-cli](https://github.com/mongodb/docs-atlas-cli/blob/master/snooty.toml#L1) this will be crucial in the future if we want to source other properties, ie. versions

Reply:
Makes sense, Seung!  I need to document the internal/external names so that both writers and stakeholders understand.